data-integration/Ballerina.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+org = "integration_tutorials"
+name = "data_integration"
+version = "0.1.0"
+distribution = "2201.7.2"
+
+[build-options]
+observabilityIncluded = true
+
+[persist]
+datastore = "mysql"
+module = "data_integration.db"
```

Comment:
Shall we use `store` as the name instead?

Reply:
Changed the package name